docs/overview_and_guide.rst

```diff
@@ -150,6 +151,16 @@ After account creation, launch Raiden with the path of your keystore supplied::
 
     raiden --keystore-path ~/.local/share/io.parity.ethereum/keys/test
 
+.. _using_rpc-endpoint:
+
+Using --eth-rpc-endpoint/Infura
+*******************************
+
+We describe using Raiden with the rpc-endpint provided by Infuras Ethereum node. Sign up with
```

Comment:
The first sentence is a bit weird to me. Can you reformulate it? I would maybe just say something like: "To use Raiden with with an rpc-endpoint provided by an Infura Ethereum node sign up with `Infura <link to sign up>`_ to get an API token. After that...."